src/blocktypes/data.nix

```diff
@@ -25,9 +25,14 @@
     }: let
       inherit (nixpkgs.legacyPackages.${system}) pkgs;
 
+      # if target ? __std_data_wrapper, then we need to unpack from `.data`
       json = pkgs.writeTextFile {
         name = "data.json";
-        text = builtins.toJSON target;
+        text = builtins.toJSON (
+          if target ? __std_data_wrapper
+          then target.data
+          else target
+        );
```

Comment:
I see – re: my most recent comment: I think this difference in structure is responsible for the error I ran into.